search/api.py

```diff
@@ -129,3 +163,14 @@ def reformat(key, values, missing_count=None):
         ret_dict[key] = reformat((key, label), values)
 
     return ret_dict
+
+
+def get_vocab_name(vocab_id):
+    """Get vocabulary name by ID."""
+    vocab = Vocabulary.objects.get(id=vocab_id)
+    return vocab.name
+
+
+def get_term_label(term_id):
+    """Get term label by ID."""
+    return Term.objects.get(id=term_id).label
```

Comment:
This is not a problem of this PR, but I guess we should store these infos in elasticsearch as well and avoid to hit the DB